mayastor/src/bdev/nexus/nexus_bdev_children.rs

```diff
@@ -440,16 +440,16 @@ impl Nexus {
     /// Add a child to the configuration when an example callback is run.
     /// The nexus is not opened implicitly, call .open() for this manually.
     pub fn examine_child(&mut self, name: &str) -> bool {
-        for mut c in &mut self.children {
-            if c.name == name && c.state == ChildState::Init {
+        self.children
+            .iter_mut()
+            .filter(|c| c.state == ChildState::Init && c.name == name)
+            .any(|c| {
                 if let Some(bdev) = Bdev::lookup_by_name(name) {
-                    debug!("{}: Adding child {}", self.name, name);
                     c.bdev = Some(bdev);
```

Comment:
Was removing the debug message intentional?

Reply:
I considered the message to be redundant. We have a message that says that a child came online in nexus_module.rs:126. If the nexus is able to open, it will show in the logs, and if it can't a message that its incomplete is added. This is only visible, however, when we debug messages are enabled.

Reply:
Ok,